baseapp-chats/baseapp_chats/graphql/object_types.py

```diff
@@ -96,9 +103,7 @@ class BaseChatRoomObjectType:
     participants = DjangoConnectionField(get_object_type_for_model(ChatRoomParticipant))
     unread_messages_count = graphene.Int(profile_id=graphene.ID(required=False))
     image = ThumbnailField(required=False)
-
-    def resolve_all_messages(self, info, **kwargs):
-        return self.messages.all().order_by("-created")
+    is_archived_by_current_profile = graphene.Boolean()
```

Comment:
this is nice to have the `profile_id` as input as well, like we have for `unread_messages_count`, the `current_profile` is a temporary hack, we should have as a goal to always send the profile_id to the query.

maybe lets name it `is_archived` to make it cleaner